app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/ColorCache.java

```diff
@@ -70,7 +70,10 @@ public enum X {
         color_smb_line("color_smb_line"),
         color_number_wall("color_number_wall"),
         color_number_wall_shadow("color_number_wall_shadow"),
-        color_basal_tbr("color_basal_tbr");
+        color_basal_tbr("color_basal_tbr"),
+        textcolor_high_values("textcolor_high_values"),
+        textcolor_inrange_values("textcolor_inrange_values"),
+        textcolor_low_values("textcolor_low_values"),;
```

Comment:
Please can you put the semicolon on a new line? In that case a future diff would be smaller. 

Also, is the color limited to text? The arrow is also changed (indeed, it is a Unicode character). If the prefix 'text' is removed, it would be in line with the other colors and the usage can be generalized.

Reply:
color_[high/inrange/low]_values is already taken for graph colors. Can't really think of better names. Suggestions?